### ISSUE TITLE
Argument parsing

### ISSUE THREAD

**Issue by ariG23498** [terrencepreilly/darglint#158]
With the google doc style I try writing a docstring in this format
```python
def model(a,b):
    """
    This is a model

    Args:
        a (int): This is the step
        b (float): This is the learning rate
    
    Returns:
        The multiplication of a and b
    """
    return a*b
``` 

I think the right task would be to throw an error stating that arguments must always be followed by a `colon` and not space. This check could be vital for an automatic documentation generation that solely depends on the position of the `colon`. Could there be a quick fix to this?

**Comment by terrencepreilly** [comment]
Are you saying you don't think it should allow type annotations in the docstring? (Like, you think your first argument should be `a: this is the step` and not `a (int): This is the step`?)  Type annotations are actually [required](https://google.github.io/styleguide/pyguide.html#doc-function-args) by the style guide if they're missing from the docstring.  So disallowing them isn't something I'd support.

Feel free to correct me in a comment.  For now, I'll close this.

**Comment by ariG23498** [comment]
So I think that we should provide type annotations but should have a way to also know where the colon lies.
`a: (int) This is a step`.
I would like to know if there is a workaround to let the linter throw an error upon where the error is thrown.

**Comment by terrencepreilly** [comment]
The format for type annotations are taken from the [napolean extension for sphinx](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), and it also seems to be the most widely used format.

If you have some particular use-case in mind, you're welcome to fork the repository and update the parser.  It shouldn't be too difficult.
